tests/lints/get-current-task.rs

```diff
@@ -62,11 +63,64 @@ fn no_match_different_signature() {
     let code = indoc! { r#"
         task = (struct task_struct *)bpf_get_current_task("foobar");
     "# };
-    assert_eq!(lint_report(code), "");
+    assert_eq!(lint_report(code, None), "");
 
     // TODO: This construct should actually be accepted. Sigh.
     let code = indoc! { r#"
         task = (struct task_struct *)bpf_get_current_task(/* WRONG */);
     "# };
-    assert_eq!(lint_report(code), "");
+    assert_eq!(lint_report(code, None), "");
+}
+
+// Test where the user kernel is greater than or equal to user specified kernel
```

Comment:
It's fine to have this for one lint, but I don't believe testing it for so many adds any value. If anything, we should probably have a dedicated test with a bunch of user input kernel and variable minimum version specifications using our custom lint https://github.com/d-e-s-o/bpflint/blob/fbb17d390373400b1863a3e06a54c47edf97057b/src/lint.rs#L269